torchvision/utils.py

```diff
@@ -223,6 +223,12 @@ def draw_bounding_boxes(
 
     colors = [(ImageColor.getrgb(color) if isinstance(color, str) else color) for color in colors]
 
+    if font is None and font_size is not None:
+        warnings.warn("Argument 'font_size' will be ignored since 'font' is not set.")
+        font_size = None
+    elif font is not None and font_size is None:
+        font_size = 10
+
```

Comment:
Will it be nice to club this code along with line 245? So that we handle font and font size in single place?

No strong opinion. 
What do you both think?
cc @pmeier

Reply:
Yes, that is a really good idea. Added a suggestion for this in https://github.com/pytorch/vision/pull/5785/files#r850126056.